samase.py

```diff
@@ -1,34 +1,19 @@
 import os
 import subprocess
-import enum
 
-SAMASE = "bin\\samase.exe"
-SAMASE64 = "bin\\samase64.exe"
+def internal_run(args: list, is64: bool):
+    SAMASE = "bin\\samase.exe"
+    SAMASE64 = "bin\\samase64.exe"
 
-class OpType(enum.Enum):
-    Run = enum.auto(), # ex: samase.exe C:\samase\uedaip
-    Build = enum.auto(), # ex: samase.exe C:\samase\uedaip --pack C:\samase\UEDAIP_BroodWar.exe --pack_version "23.2212" --pack_icon uedr.ico
-    Unpack = enum.auto(), # ex: samase.exe C:\samase\unpack --unpack C:\samase\unpack.exe
+    path = [SAMASE if not is64 else SAMASE64, *args]
+    print(f"Samase string debug: {path}")
+    subprocess.run([path, *args])
```

Comment:
This internal run func is probably confusing due to `*args`, but what it basically means it should pass the elements of a list as if they were passed literally as arguments. 

e.g. for `args = [1,2,3]` `print(*args)` means `print(1, 2, 3)`